Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/deploymentslot/DeploymentSlotNodePresenter.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) Microsoft Corporation
+ *
+ * All rights reserved.
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+ * the Software.
+ *
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.microsoft.tooling.msservices.serviceexplorer.azure.webapp.deploymentslot;
+
+import java.io.IOException;
+
+import com.microsoft.azuretools.core.mvp.ui.base.MvpPresenter;
+
+public class DeploymentSlotNodePresenter<V extends DeploymentSlotNodeView> extends MvpPresenter<V> {
+    public void onStartDeploymentSlot(String subscriptionId, String webAppId, String slotName) throws IOException {
+        //todo
+    }
+
+    public void onStopDeploymentSlot(String subscriptionId, String webAppId, String slotName) throws IOException {
+        //todo
+    }
+
+    public void onRestartDeploymentSlot(String subscriptionId, String webAppId, String slotName) throws IOException {
+        //todo
+    }
+
+    public void onNodeRefresh() {
```

Comment:
onRefreshNode

Reply:
Seems the previous patterns all look like on**NoneVerb**, like the `onModuleRefresh`, `onStartWebApp`.

Reply:
`onRefreshNode` would be better, see `onStartDeploymentSlot`, `onSwapWithProduction` ..etc in this file